clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -464,8 +464,15 @@ public void testQuotaMetrics() throws Exception {
 
         // normal fetch
         for (int i = 1; i < 4; i++) {
+            // We need to make sure the message offset grows.
```

Comment:
the comment doesn't make it clear why we need to make sure the message offset grows